Changelog.md

```diff
@@ -1,4 +1,8 @@
 # Changelog
+
+## 2.0.8
```

Comment:
So it looks like we haven't even shipped 2.0.7 yet (the pipeline needs to be run manually) so this PR doesn't need to have its version bumped.